spec/factories/factory.rb

```diff
@@ -7,4 +7,9 @@
     sequence(:email) { |n| "user#{n}@launch.com" }
     password "SuperPWthatislong"
   end
+
+  factory :lifehack do
```

Comment:
I forget if I told you or someone else this before, but eventually you might want to break all your factories out into one per file for ease of use.